src/main/java/com/solvd/api/BaseRestAssuredMethod.java

```diff
@@ -2,84 +2,47 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.solvd.enums.HttpMethodTypeEnum;
+import com.solvd.enums.HttpMethod;
 import com.solvd.factory.ObjectMapperFactory;
-import com.solvd.graphql.GraphQlQuery;
 import com.solvd.utils.FileReader;
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
 import io.restassured.response.Response;
 import io.restassured.specification.RequestSpecification;
 
 public abstract class BaseRestAssuredMethod {
-    private static final String BEARER_TOKEN = FileReader.getConfigValue("token");
-    private static final ObjectMapper objectMapper = ObjectMapperFactory.buildNew();
-    public String endpointUrl;
+    protected static final String BEARER_TOKEN = FileReader.getConfigValue("token");
+    protected static final ObjectMapper objectMapper = ObjectMapperFactory.buildNew();
```

Comment:
static BEARER_TOKEN and ObjectMapper means that all services will use the same token and mapper.
Let's make them non-static, instantiated, pass the token through the constructor